### DISCUSSION TITLE
Embed Products and Categories like Shopify

### DISCUSSION THREAD

**Discussion by mcconnon12** [swellstores/community#286] [Ideas]
I was just curious if this feature is on the roadmap...

Shopify has had the ability to do product and category embeds for a few years. 

This makes integration to existing sites seamless and fast. 

Purchases links are just too limited. 

Here's a short video: https://www.youtube.com/watch?v=sTINU1DE4HE

I think Swell could tap into a lot of small businesses who don't want to change their existing site or use integrations they are not familiar with.

**Comment by ericingram** [comment]
This is a great idea. It’s not in the roadmap yet but as we are building out Swell Apps, that could be a good middle ground.

Our goal is to make this stack available and easy to use for small businesses, but it will take some more time to get there.

**Reply by mcconnon12** [reply]
Hey Eric, yea seems like low hanging fruit here. This would be the fastest and easiest way for businesses to integrate Swell. I can only imagine how many more businesses would be trying Swell if they had an "easy" solution to get started. Swell has one of the best subscription management platforms out there but the road block is integrating into an existing stack.

**Reply by ericingram** [reply]
I agree completely! We will get there soon, just watch :)

**Reply by mcconnon12** [reply]
Awesome! It would make Swell epic. I tried a ton of platforms and the subscription control in Swell is one of the best I've scene.

**Reply by mcconnon12** [reply]
@ericingram another thing that goes along with this for an easier transition would be the ability to embed the client portal or do a cname login link that does not include the store since that is going away. Is there any timeframe where the embedded products/categories could happen? Thanks.

**Reply by ericingram** [reply]
We probably won't prioritize this or any major feature until after Swell Apps launch, which is likely early next year

**Reply by mcconnon12** [reply]
Got ya. Maybe in the meantime there could be some kind of work around. We kind of pieced something together but it's disjointed with the client portal being tied to the store. Any thoughts?

**Reply by ericingram** [reply]
I think can app would be a good way to solve that in the near future. Nothing else comes to mind currently

**Comment by mcconnon12** [comment]
Hey Swell Team,

I wanted to circle back and see if embedding products and/or categories is on the radar?

Thanks,
Shawn

**Reply by Ghostess13** [reply]
So, I haven't been able to use Swell in production since the storefronts were deprecated. I asked about embedding buy buttons and checkout a few years ago. Still waiting.....Hoping that this feature will arrive before the holiday shopping season. I would like to use the 2 stores that have been shelfware for quite some time.

**Reply by ericingram** [reply]
We’ve been working on a new storefront release. It will be much better than the V1. Announcement going out soon.

**Reply by mcconnon12** [reply]
That's great to hear. What is the ETA?

**Reply by ericingram** [reply]
Announcement in the next week or two and we'll have a discord channel for Q&A. Beta should be available in October for early signups. Talk to support to request earlier access

**Reply by mcconnon12** [reply]
Hey Eric,

Just wanted to touch base on this...

Do you have a link to the Discord channel on this?

Thanks,
Shawn

**Reply by ericingram** [reply]
https://discord.com/channels/1001781312899796994/1285374808116297728
